production/training/docker/startup.sh

```diff
@@ -0,0 +1,17 @@
+set -x
+
+onEnd() {
+  kill -SIGTERM "$TOKENIZER_PID"
+  exit 0
+}
+
+# Run tokenizer in background
+cd /home/source/almond-tokenizer
+PORT=8888 LANGUAGES=en ./run.sh &
+TOKENIZER_PID=$!
+trap onEnd SIGTERM SIGINT
+sleep 5
+
+# Run daemon server
+cd /home/workdir
+/usr/bin/node /home/source/almond-cloud/training/daemon.js
```

Comment:
What user does this run as? Should we create a dedicated user for the daemon?
(The daemon has no privilege dropping code, because it's expected to be jailed from outside, e.g. using systemd)

Reply:
The default is root. Dedicated user is a good idea.  What username do you use for the current service.

Reply:
almond-training